pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```diff
@@ -38,30 +56,20 @@ stdenv.mkDerivation (finalAttrs: {
     s
     servant
     shell-split-string
-  ]);
-
-  strictDeps = true;
-
-  buildPhase = ''
-    runHook preBuild
+  ];
 
-    emacs --batch -L . -f batch-byte-compile cask.el cask-cli.el
-
-    runHook postBuild
-  '';
+  doCheck = true;
 
-  installPhase = ''
-    runHook preInstall
+  strictDeps = true;
```

Comment:
Hmm, there is no `*Inputs`, such as `nativeBuildInputs`,  set here. So `strictDeps` is a no op.  But I guess it does not hurt anyway.

Reply:
`strictDeps` does not affect it indirectly?
`melpaBuild` even generates a `nix-support` directory...